src/store/sagas/sagasActions.ts

```diff
@@ -65,3 +63,27 @@ export type makeDefaultType = {
   payload: string;
   type: typeof MAKE_DEFAULT;
 };
+
+export const deleteItem = (payload: {
+  id: string;
+  title: string | undefined;
+}): deleteItemType => ({
```

Comment:
Please, create a separate type for this and use `title?: string` instead of `undefined`